src/actions/emojis.js

```diff
@@ -133,7 +143,8 @@ export function getCustomEmojis(page = 0, perPage = General.PAGE_SIZE_DEFAULT, s
 
         if (loadUsers) {
             const usersToLoad = {};
-            Object.values(data).forEach((emoji) => {
+            Object.keys(data).forEach((key: string) => {
```

Comment:
Is this a performance-related change or something Flow imposes/makes harder?

Reply:
In this case is flow related, flow have problem to infer Object.values result data type, so this is a workaround.